includes/wikia/tests/HeliosHelperControllerTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+use Wikia\Helios\HelperController;
+
+class HeliosHelperControllerTest extends WikiaBaseTest {
+
+	public function test_shouldFailWithInvalidSecret() {
+		$response = $this->app->sendRequest( 'Wikia\Helios\HelperController', 'sendPasswordResetLinkEmail', [] );
+		$this->assertEquals( \WikiaResponse::RESPONSE_CODE_FORBIDDEN, $response->getCode() );
+		$this->assertFalse( $response->getData()['success'] );
+		$this->assertEquals( 'invalid secret', $response->getData()['message'] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "message" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)